index.bs

```diff
@@ -72,74 +73,83 @@ dictionary CSSStyleSheetInit {
-			set |sheet|'s <a spec=cssom>alternate flag</a>.
-		5. If the {{CSSStyleSheetInit/disabled}} attribute of |options| is true,
+			Otherwise, [=serialize a media query list=] from the attribute and then [=create a MediaList object=]
+			from the resulting string and set it as |sheet|'s <a spec=cssom>media</a>.
+		3. If the {{CSSStyleSheetInit/disabled}} attribute of |options| is true,
 			set |sheet|'s <a spec=cssom>disabled flag</a>.
-		6. Return |sheet|.
+		4. Return |sheet|.
 	</dd>
 </dl>
 
 <a interface>CSSStyleSheet</a> instances have the following associated states:
 <dl>
-	<dt><dfn  for=CSSStyleSheet lt="constructed flag">constructed flag</dfn></dt>
+	<dt><dfn for=CSSStyleSheet lt="constructed flag">constructed flag</dfn></dt>
 	<dd>
 		Specified when created. Either set or unset. Unset by default.
-		Signifies whether this stylesheet is made via constructor or not, so must be set only for stylesheets that are constructed using the {{CSSStyleSheet()}} or {{CSSStyleSheet(options)}} function.
+		Signifies whether this stylesheet was created by invoking the IDL-defined constructor.
 	</dd>
 	<dt><dfn for=CSSStyleSheet lt="disallow modification flag">disallow modification flag</dfn></dt>
 	<dd>
-		Either set or unset. Unset by default. If set, modification to the stylesheet's rules are not allowed.
+		Either set or unset. Unset by default. If set, modification of the stylesheet's rules is not allowed.
 	</dd>
 	<dt><dfn for=CSSStyleSheet lt="constructor document">constructor document</dfn></dt>
 	<dd>
-		Specified when created. The {{Document}} where the stylesheet is originally constructed on. Null by default.
+		Specified when created. The {{Document}} a constructed stylesheet is associated with. Null by default.
+		Only non-null for stylesheets that have [=constructed flag=] set.
 	</dd>
 </dl>
 
 Modifying Constructed Stylesheets {#modifying-constructed-stylesheets}
 =============================
 
-After construction, constructed stylesheets can be modified using rule modification methods like <a spec=cssom-1 for="CSSStyleSheet">insertRule(rule[, index])</a> or <a spec=cssom-1 for="CSSStyleSheet">deleteRule(index)</a>, or {{replace(text)}} and {{replaceSync(text)}} if the sheet's [=disallow modification flag=] is not set. If those methods are called when the sheet's [=disallow modification flag=] is set, or <a spec=cssom-1 for="CSSStyleSheet">insertRule(rule)</a> is used to add an <a>@import</a> rule, a "{{NotAllowedError}}" {{DOMException}} will be thrown as detailed in the below algorithms. 
+After construction, constructed stylesheets can be modified using rule modification methods like [[cssom/#dom-cssstylesheet-insertrule|insertRule]] or [[cssom/#dom-cssstylesheet-deleterule|deleteRule]], or {{replace(text)}} and {{replaceSync(text)}}. The behavior of those methods are affected by the [=disallow modification flag=] and [=constructed flag=] as described below.
```

Comment:
This isn't the linking syntax you want. `{{CSSStyleSheet/insertRule()}}` will correctly link to the method. (You only need to supply arglists if the method has multiple overrides; if not, you can always autolink it with just `()`.)

Reply:
Oh, thanks!